cloud_launch_small.json

```diff
@@ -1,15 +1,31 @@
 {
-  "template": "small",
+  "template": "small_entity_db_v2",
   "world": {
     "chunkEdgeLengthMeters": 50,
     "snapshots": {
       "snapshotWritePeriodSeconds": 0
     },
     "dimensions": {
-      "xMeters": 500,
-      "zMeters": 500
+      "xMeters": 300,
+      "zMeters": 300
```

Comment:
And this one too?

Reply:
Small world has 4 layers ==> (4*2*36 + 4) ==> 292m edge length

so the spatial world only needs to be 300x300